desktop_version/src/Game.cpp

```diff
@@ -725,7 +726,8 @@ void Game::crewmate_textbox(const int r, const int g, const int b)
     graphics.addline("");
     graphics.textboxprintflags(PR_FONT_INTERFACE);
     graphics.textboxcentertext();
-    graphics.textboxpad(5, 2);
+    float spaces_per_8 = font::len(PR_FONT_INTERFACE, " ")/8.0;
```

Comment:
You are mixing `float`s and `double`s here and it's not clear to me why you are doing so.

A decimal literal defaults to type `double` and must be suffixed with `f` to be of type `float`. You seem to be assigning a `double` literal to a `float` variable. Given that on the very next line, you're passing the variable to `SDL_ceil`, whose argument takes a `double`, you should either...

1. Change the `8.0` literal to `8.0f` and change the `SDL_ceil` calls to `SDL_ceilf`, or
2. Change the type of `spaces_per_8` to `double`.

Given that it doesn't seem like `double` precision is necessary here, I'm inclined to say you should go with (1).

Reply:
Done and force-pushed.